saas/api/metrics.py

```diff
@@ -60,6 +64,40 @@ def get(self, request, *args, **kwargs): #pylint: disable=unused-argument
         return Response(result)
 
 
+class RevenueAPIView(ProviderMixin, APIView):
+    """
+    Produce revenue stats
+    """
+    def get(self, request, *args, **kwargs):
+        ends_at = request.GET.get('ends_at', None)
+        if ends_at:
+            ends_at = parse_datetime(ends_at)
+        ends_at = datetime_or_now(ends_at)
+
+        table_key = request.GET['table_key']
```

Comment:
Won't that throw a 500 error on a GET request without a ``table_key`` query parameter?